### PR TITLE
Added buttons for syncing

### PR DESCRIPTION
**Pull request by MayurLund**
Issue #387 

Overview:
Added 2 buttons for syncing the data for discord users data and unverified Discord users data. The buttons are shown only to super user.
screen-capture (1)

The second button is failing as I don't have discord setup but I have written the logic for it so please check that out. 

The test cases are yet to be written for the above work.

### CODE REVIEW

File: index.html
[the remark this answers is not in the document]
Comment (MayurLund) [reply index.html:1]: Hey @sahsisunny , Thanks for pointing out. I just noticed that now and moreover in the remaining file both the cases are used and its not consistent. So for now I will keep it to kebab case as majority of the code is in kebab case. To make the previous code casing consistent, should I make changes in this PR or create a new issue for that?

**Comment by gauravsinhaweb** [comment]
@MayurLund  It would be nice if you could attach a working video of the implementation, and also provide a short description in the PR of what changes/ functionality you made in this PR

File: index.html
[the remark this answers is not in the document]
Comment (MayurLund) [reply index.html:1]: Hey @sahsisunny Resolved that for my code. Please have a look at it.

**Review by heyrandhir — CHANGES_REQUESTED** [review]

**Review by gauravsinhaweb — APPROVED** [review]

File: script.js [hunk 1]
```diff
@@ -20,12 +32,91 @@ export async function showSuperUserOptions(...privateBtns) {
...
+  extensionRequestsLink,
+  syncDiscordUserDataButton,
+  syncUnverifiedUserDataButton,
+  syncDiscordUserDataStatus,
+  syncUnverifiedDiscordStatus,
+);
 
 const createGoalButton = document.getElementById('create-goal');
 const params = new URLSearchParams(window.location.search);
 if (params.get('dev') === 'true') {
```
[the remark this answers is not in the document]
Comment (MayurLund) [reply script.js:46]: Hi @gauravsinhaweb I have not made these changes. Someone tried to test their functionality and pushed the changes too. So I am not making changes to those lines as they were previously present.

**Review by sahsisunny — APPROVED** [review]
Please link the ticket for test

**Review by heyrandhir — DISMISSED** [review]

**Comment by MayurLund** [comment]
> Please link the ticket for test
Created a new issue https://github.com/Real-Dev-Squad/website-dashboard/issues/422 for tests and linked up with the PR

**Review by prakashchoudhary07 — APPROVED** [review]

**Pull request merged**